### PR TITLE
Fix naming in TF MobileBERT

### LINKED ISSUE #10088: Language modelling head has zero weights in pretrained TFMobileBertForMaskedLM
## Description
The `TFMobileBertForMaskedLM` example returns all zero logits while `MobileBertForMaskedLM` example works fine.
https://huggingface.co/transformers/model_doc/mobilebert.html#tfmobilebertformaskedlm

I checked language modeling head weights for both models and found that TF pretrained model has zero weights.

## Environment info

- `transformers` version: 4.3.0
- Platform: Google Colab
- Python version: 3.6.9
- PyTorch version (GPU?): 1.7.0+cu101
- Tensorflow version (GPU?): 2.4.1
- Using GPU in script?: No
- Using distributed or parallel set-up in script?: No

### Who can help
@jplu @patrickvonplaten

## Information

Model I am using (Bert, XLNet ...): TFMobileBertForMaskedLM

The problem arises when using:
- [x] the official example scripts: (give details below)

## To reproduce

```
from transformers import MobileBertForMaskedLM
from transformers import TFMobileBertForMaskedLM

# PyTorch
model = MobileBertForMaskedLM.from_pretrained('google/mobilebert-uncased')
print(list(model.cls.parameters())[0])

# Parameter containing:
# tensor([-7.2946, -7.4302, -7.5401,  ..., -7.4850, -7.4503, -2.7798],
#       requires_grad=True)

# TensorFlow
model = TFMobileBertForMaskedLM.from_pretrained('google/mobilebert-uncased')
print(model.layers[1].get_weights()[0])

# array([0., 0., 0., ..., 0., 0., 0.], dtype=float32)
```
## Expected behavior
Language modeling head for TFMobileBertForMaskedLM should have the same weights as of MobileBertForMaskedLM

### PR DESCRIPTION
**Pull request by jplu**
# What does this PR do?

This PR fixes a naming issue in the `TFMobileBertForMaskedLM` model.

# Fixes

#10088

### CODE REVIEW

**Review by LysandreJik — APPROVED** [review]
Yes, LGTM! Thanks for taking care of it @jplu
